app/javascript/components/pages/NotFound.test.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react'
+
+import Enzyme, { shallow } from 'enzyme'
+
+import Adapter from 'enzyme-adapter-react-16'
+
+import NotFound from './NotFound'
+
+import pngegg from '../assets/pngegg.png'
+
+Enzyme.configure({adapter: new Adapter()})
+
+// describe("When NotFound renders", () => {
```

Comment:
Normally i would not approve this. We should never check in commented out code like this as i explained in the slack thread. Same with console.logs or debugger statements